charts/fybrik/files/taxonomy/attributes.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Perhaps change file name to infra-attributes.json for clarity?

Reply:
The name of the file is auto-generated from the package name, and there are some lint rules for package naming. Thus, the result will be infraattributes.json